src/components/CookieBanner/CookieBanner.js

```diff
@@ -0,0 +1,35 @@
+import React, {useState} from 'react';
+import { CgClose } from "react-icons/cg";
+import CookieIcon from '../../img/cookieIcon.png'
+import VhiLogo from '../../img/cookieVhiLogo.png'
+
+export default function CookieBanner() {
+
+    const [isVisible, setIsVisible] = useState(false);
+
+    function ShowCookieBanner() {
```

Comment:
first of all the function name dont start with capital letter. 2ndly, the function should start with "on"(onClickShow)

Reply:
I have corrected this.